amida-tech/indaba-client
# Styling/inba 278 colors for dnd

#### What does this PR do?
When you drag a user over to drop, you should now see a color coding over the CSS that indicates if you can or cannot drop the user. Green for go, red for no. 

I've invited Jon to gander at the code because I don't know if these CSS modifications are acceptable. I needed to increase my specificity to override Bootstrap, but parent elements are in parent components, while the `td` was in a child. 

#### Related JIRA tickets:

#### How should this be manually tested?
Create a project with users and groups, stages, subjects and a survey.
Drag a user over to a stage of which is group is acceptable. It should appear green.
Drag a user over to a stage where his group isn't assigned. It should appear reddish.
Drag a user over another user. It should appear reddish. 

#### Background/Context
The colors, Duke! The colors!

#### Screenshots (if appropriate):
